app/src/main/java/com/halim/adam/cashmaster/AddNewBudget.java

```diff
@@ -0,0 +1,24 @@
+package com.halim.adam.cashmaster;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.EditText;
+
+public class AddNewBudget extends Activity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_add_new_budget);
+    }
+
+    public void GetInput(View view){
+        EditText inputName = findViewById(R.id.inputName);
+        EditText inputPortion = findViewById(R.id.inputPortion);
+        EditText inputAmount = findViewById(R.id.inputAmount);
+
+        DatabaseHelper dbHelper = new DatabaseHelper(this);
+        dbHelper.InsertBudget(inputName.getText().toString(), Float.valueOf(inputPortion.getText().toString()), Float.valueOf(inputAmount.getText().toString()));
```

Comment:
Styling feedbacks: If the line is too long, you can break it down to multiple lines so people will have an easier time to read